lib/introspective_grape/error_formatter/simple/camel_json.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'base'
+require_relative '../../utils/key_transformations'
+# require_relative '../../utils/json_expander'
+module IntrospectiveGrape
+  module ErrorFormatter
+    module Simple
+      module CamelJson
+        extend IntrospectiveGrape::ErrorFormatter::Simple::Base
+
+        class << self
+
+          def call(message, backtrace, options = {}, env = nil)
+            MultiJson.dump(camelize_keys(format(message, backtrace, options, env)))
```

Comment:
With the 0.3.3 release you can use CamelSnakeKeys.camel_keys here (it no longer monkey patches Enumerable) and add the error formatters without the IntrospectiveGrape::Utils changes.